_ml-commons-plugin/api/mcp-server-apis/sse-message.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: SSE Message 
+parent: Agent APIs
+grand_parent: ML Commons APIs
+nav_order: 50
+---
+
+# SSE Message 
+**Introduced 3.0**
+{: .label .label-purple }
+
+This is the standard message interaction endpoint of MCP protocol, you can use this API to send messages to MCP server in OpenSearch. Usually you don't need to explicitly interact with this API if you're using standard MCP client. You can checkout the example client here: https://github.com/zane-neo/opensearch-mcpserver-test-example
```

Comment:
Can we host this in the ML commons repo instead of your personal one?

Reply:
Need to confirm with team on this, but if you think this is not the correct approach, I can remove this part for now.

Reply:
Since this topic will not be used by most users, let's comment out the example for now until we're able to move it to one of the OpenSearch repos.